src/download_article_metadata.py

```diff
@@ -0,0 +1,48 @@
+"""
+This script uses the pubmedby package to download information about articles
+with specific MeSH headings from pubmed.
+The code is a modification of
+https://github.com/greenelab/iscb-diversity/blob/master/01.download-pubmed.ipynb
+and is used in accordance with their licenses
+"""
+
+import argparse
+import pathlib
+import lzma
+import tqdm
+
+from pubmedpy.eutilities import esearch_query, download_pubmed_ids
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('mesh_term',
+                        help='The MeSH term whose article info should be downloaded')
+    parser.add_argument('--overwrite',
+                        help="Set this flag to redownload terms that "
+                             "already have data",
+                        action='store_true')
+
+    args = parser.parse_args()
+
+    # Make the term safe for putting in a path
+    path_term = '_'.join(args.mesh_term.split(' '))
```

Comment:
Could you just do:
```suggestion
    path_term = args.mesh_term.replace(' ', '_')
```
Or could there be multiple spaces between parts of a term?

Reply:
Yeah, that would definitely make more sense